app/components/point.jsx

```diff
@@ -28,6 +29,7 @@ function Point(props) {
   })
 
   return (
+    <div class="sk_bg big">
```

Comment:
Visually, it's the subject and description fields below that should be shimmering - rather than the whole thing.
Also, there needs to be a way that it's on when the vState is 'loading' and off otherwise.